### DISCUSSION TITLE
Avatar update size and border

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1103] [Proposals]
Siguiendo el trabajo de actualización visual de la nueva Vivo APP, necesitamos en Avatar:

- Nuevo tamaño 32px.
  - Solo necesitamos crear una variante de tamaño 32 en Figma para que podamos estandarizar su uso entre los diseñadores.
- Necesitamos un borde en Avatar. Tipo on/off.
  - Borde de 1px con color #fff.

En código no habrá actualización ni cambiaremos nada ya usado en los componentes actuales.

¿Podría crear una Branch en Avatar Specs para que podamos proponer actualizaciones según lo acordado en una reunión?? 

![avatar border](https://user-images.githubusercontent.com/123083886/223184826-c99e9d18-6907-45c5-b264-46d5b0ee88e7.png)

![avatar new size](https://user-images.githubusercontent.com/123083886/223184836-c6926b88-9f4d-488a-821c-6760bd39b405.png)

Gracias

**Comment by yceballost** [comment]
Buenas Bruno!

1. Añadiremos 32px como tamaño alternativo en el componente de Figma #1107 
2. Añadiremos la posibilidad de pasarle un borde al avatar como he comentado en la discussion #1104

**Reply by BrunoBiagioniNeto** [reply]
Poderia crear la Branch en Avatar Specs? Ya tenemos el componente listo e ya lo pasamos para el equipo del desarrolho.

**Comment by annefranca** [comment]
@yceballost 

Adicionei a borda de 1px na documentação.

[#1108 - Avatar border](https://www.figma.com/file/U4ipIXOk64bdM5tSvaqPKS/branch/iho0A0GNr7c8tri1JDBl7f/%F0%9F%94%B8-Avatar-Specs?node-id=2%3A61&t=p1FtKS1555Zx92Xu-1)
